### PR TITLE
Update release notes for v6 24 00

### LINKED ISSUE #7858: Missing in Release Notes for 6.24
List of relevant PR missing in the release notes for 6.24

- XROOTD  #7380 
- TMVA : optimizer options : #7318
- ChangeLabel: #7294
- Buffer Merger improvements : #7286
- Eve7: The first implementation of log view #7270
- TMVA: Update Keras tutorials using tensorflow.keras #7255
- builtins:  Zstd 1.4.8 #7214
- RooFit: warning message for RooAddPdf without norm set: #7129 
- RooFit: Improve plotting of RooBinSamplingPdf. #7110
- Hist: Fixes and improvements for the TKDE class #7096
- Fix a memory leak in TF1::Copy #7035
- Fix for new version of cudnn version 8 #7003
- Minuit2: Use Numerical gradient as input to the seed generator when using external gradients in Minuit2 #7001
- TF1: Enable using Automatic differentiation for computing Parameter Gradient in TF1 #6956
- Math: Fix correction factor for Confidence level in ROOT::Fit::FitResult #6902
- TMVA: Implement MethodCategory::GetMulticlassValues() #6886
- Hist: Add THistRange class defining a generic histogram range and its iterator #6795
- TMVA: Support  for Keras 2.4 (tf.keras) #6793
- Eve-7: Add example of calorimeter representation of REveDataCollection #6783
- Math: Fix BinData copy constructor #6776
- [tmva] Add possibility to make ROC curve for the training dataset from TMVA::Factory #6724
- Math: Changes in TRandom3::GetSeed  #6723
- Minuit2: new Minuit2 logging #6705
- ntuple] Several interface improvements (unique pointer, const) #6655
- Optimize TPrincipal::AddRow function #6630

### PR DESCRIPTION
**Pull request by lmoneta**
Update release notes with missing items, fixes #7858

### CODE REVIEW

File: README/ReleaseNotes/v624/index.md [hunk 1]
```diff
@@ -194,13 +201,33 @@ Tests for the Spark backend can be turned ON/OFF with the new build option `test
 - Due to some planned major improvements to `RVec`, the layout of `RVec` objects will change in a backward-incompatible way between v6.24 and v6.26.
   Because of this, we now print a warning if an application is reading or writing a `ROOT::RVec` object from/to a ROOT file. We assume this is an
   exceedingly rare case, as the ROOT interface typically used to manipulate `RVec`s is `RDataFrame`, and `RDataFrame` performs an on-the-fly
-  `RVec <-> std::vector` conversion rather than writing `RVec`s to disk. Note that, currently, `RVecs` written e.g. in a `TTree` cannot be read back
+  `RVec <-> std::vector` conversion rather than writing `RVec`s to disk. Note that, currently, `RVecs` written e. Enable autig. in a `TTree` cannot be read back
```
Comment (Axel-Naumann) [inline README/ReleaseNotes/v624/index.md:204]: This seems broken?
```suggestion
  `RVec <-> std::vector` conversion rather than writing `RVec`s to disk. Note that, currently, `RVecs` written e.g. in a `TTree` cannot be read back
```

File: README/ReleaseNotes/v624/index.md [hunk 2]
```diff
@@ -194,13 +201,33 @@ Tests for the Spark backend can be turned ON/OFF with the new build option `test
...
   using certain ROOT interfaces (e.g. `TTreeReaderArray`, `RDataFrame` and the experimental `RNTuple`). All these limitations will be lifted in v6.26.
 - Portable implementation of the RANLUX++ generator, see [RanluxppEngine](https://root.cern/doc/master/classROOT_1_1Math_1_1RanluxppEngine.html) and [our blog post](https://root.cern/blog/ranluxpp/).
+- Change `TRandom3::GetSeed` to return the current state element in the contained seed vector of TRandom3. The return value will now change after every call of `TRandom3::Rndm` (when generating a random number). Before the function was returning the first element of the state, which was changing only after 624 calls to `Rndm()`.
+- Fix a bug in `ROOT::Fit::BinData` copy constructor
+- Fix a bug in applying a correction factor used for the computation of the fit confidence level in `ROOT::Fit::FitResult`.
+- TMatrix: optimize implementation of `TPrincipal::AddRow` that is heavily used by CMS.
+
+###Minuit2
```
Comment (Axel-Naumann) [inline README/ReleaseNotes/v624/index.md:212]:
```suggestion
### Minuit2
```

File: README/ReleaseNotes/v624/index.md [hunk 3]
```diff
@@ -215,6 +242,9 @@ Tests for the Spark backend can be turned ON/OFF with the new build option `test
   These points will be skipped, and HypoTestInverter can continue.
 - Tweak pull / residual plots. ROOT automatically zoomed out a bit when a pull / residual plot is created. Now, the
   axis range of the original plot is transferred to the residual plot, so the pulls can be drawn below the main plot.
+- Improve plotting of `RooBinSamplingPdf`
+- Print a Warning message when the `RooAddPdf` is evaluated without passing a normalization set and the class has not a normalization set defined.
+Without a normalization set the `RooAddPdf` is not properly defined and its shape will be different depending on which normalization range is used.
```
Comment (Axel-Naumann) [inline README/ReleaseNotes/v624/index.md:247]:
```suggestion
 Without a normalization set the `RooAddPdf` is not properly defined and its shape will be different depending on which normalization range is used.
```

File: README/ReleaseNotes/v624/index.md [hunk 4]
```diff
@@ -194,13 +201,33 @@ Tests for the Spark backend can be turned ON/OFF with the new build option `test
...
+- Add a new improved message logging system. Debug message now can be enabled in Minuit2 when using maximum print level.
+- When using external provided gradient, compute in MnSeed still numerical gradients to obtain correct step sizes and  initial estimate of covariance matrix. This allows to start with a good first
+	state estimation, reducing significantly the number of  subsequent iterations.
+
 
 ## TMVA
 
-- Introducing TMVA PyTorch Interface, a method to use PyTorch internally with TMVA for deep learning. This can be used as an alternative to PyKeras Interface for complex models providing more flexibility and power.
+- Introducing TMVA PyTorch Interface, a method to use PyTorch internally with TMVA for deep learning. This can be used as an alternative to PyKeras Interface for complex models providing more
+flexibility and power.
```
Comment (Axel-Naumann) [inline README/ReleaseNotes/v624/index.md:222]:
```suggestion
- Introducing TMVA PyTorch Interface, a method to use PyTorch internally with TMVA for deep learning. This can be used as an alternative to PyKeras Interface for complex models providing more flexibility and power.
```

File: README/ReleaseNotes/v624/index.md [hunk 5]
```diff
@@ -380,7 +410,8 @@ The new `RooCrystalBall` class can substitute the `RooDSCBShape` and `RooSDSCBSh
 
 - Add the method `AddPoint`to `TGraph(x,y)` and `TGraph2D(x,y,z)`, equivalent to `SetPoint(g->GetN(),x,y)`and `SetPoint(g->GetN(),x,y,z)`
 - Option `E0` draws error bars and markers are drawn for bins with 0 contents. Now, combined
-  with options E1 and E2, it avoids error bars clipping.
+with options E1 and E2, it avoids error bars clipping.
```
Comment (Axel-Naumann) [inline README/ReleaseNotes/v624/index.md:413]:
```suggestion
  with options E1 and E2, it avoids error bars clipping.
```
Else this ends the enumeration

**Review by Axel-Naumann — CHANGES_REQUESTED** [review]
